### DISCUSSION TITLE
Can not correctly paste multi line SQL statement under the sqlplus tool through XTERMJS

### DISCUSSION THREAD

**Discussion by rysinal** [sourcelair/xterm.js#4372] [Q&A] [answered]
I used the clipboard to paste the same SQL statement, and it pasted normally under vi editor, but under sqlplus there was an unexpected number at the end, I guess this number is the line number

```
select owner,table_name,
num_rows
from dba_tables
```

## Details
- Browser and browser version: Chrome 108.0.5359.124
- OS version: MacOS 12.6
- xterm.js version: 4.14.1

### Steps to reproduce

![sqlfix](https://user-images.githubusercontent.com/9962863/212831029-66d9f92a-35e3-4e8d-9f78-cfcf30ffcc40.gif)

**Comment by jerch** [comment, marked as answer]
Most cmdline REPLs use NL as a command separator, thus cannot be fed from multiline commands directly. Most likely you'd have to resemble the command into one big line, or into smaller pieces. Yes the numbers look like sqlplus answers line input with a line number, but since you entered the lines "at once", the screen state gets scrambled and numbers are appended at the end.
This is not an issue of the terminal (as you showed with vim), but how your app tries to interact with you. If anything, ask in sqlplus help forums.
